.github/workflows/cut-new-release.yml

```diff
@@ -0,0 +1,100 @@
+name: Cut a new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: Version of the next release (e.g. v0.15.0)
+        required: true
+
+env:
+  git-user: github-actions[bot]
+  git-email: 41898282+github-actions[bot]@users.noreply.github.com
+  main-branch: main
+
+jobs:
+  workflow-metadata:
+    name: Extract workflow metadata
+    runs-on: ubuntu-22.04
```

Comment:
any reason to not use `ubuntu-latest`? I know that latest points to latest LTS so `ubuntu-20.04` but do we need newer? in case of security issues we won't be automatically "updated"

Reply:
Once we provide the latest, there would be a silent upgrade open to unexpected errors. I just wanted to be on the safe side.